java/ql/lib/semmle/code/java/UnitTests.qll

```diff
@@ -125,6 +125,53 @@ class JUnitJupiterTestMethod extends Method {
   }
 }
 
+/**
+ * A JUnit 5 test method.
+ * A test method is defined by JUnit as "any instance method
+ * that is directly annotated or meta-annotated with `@Test`,
+ * `@RepeatedTest`, `@ParameterizedTest`, `@TestFactory`, or
+ * `@TestTemplate`."
+ * See https://junit.org/junit5/docs/current/user-guide/#writing-tests-definitions
```

Comment:
If you want this to be three paragraphs, you need a blank line between each paragraph. If you hover over any `hasLocationInfo` you'll see that a single line break is ignored when formatting the QLDoc. (Same for the class below.)
```suggestion
 * A JUnit 5 test method.
 *
 * A test method is defined by JUnit as "any instance method
 * that is directly annotated or meta-annotated with `@Test`,
 * `@RepeatedTest`, `@ParameterizedTest`, `@TestFactory`, or
 * `@TestTemplate`."
 *
 * See https://junit.org/junit5/docs/current/user-guide/#writing-tests-definitions
```

Reply:
Updated in [faeb7ab](https://github.com/github/codeql/pull/19094/commits/faeb7ab8909eefb087cdc1cf235506c875bbf451).